memgpt/memory.py

```diff
@@ -124,7 +126,10 @@ def core_memory_replace(self, name: str, old_content: str, new_content: str) ->
         Returns:
             Optional[str]: None is always returned as this function does not produce a response.
         """
-        self.memory[name].value = self.memory[name].value.replace(old_content, new_content)
+        if old_content == "":
```

Comment:
Maybe give it a hint like "Use `core_memory_append` to add new content without replacing any existing content." to give to the model?

Reply:
I updated the ValueError that it gets